mlserver/parallel/pool.py

```diff
@@ -117,7 +119,10 @@ async def predict(
             model_version=model_version,
             inference_request=inference_request,
         )
-        self._requests.put(request_message)
+
+        worker_pid = next(self._workers_round_robin)
+        worker = self._workers[worker_pid]
```

Comment:
could we abstract this bit so that we have different "load balancing" in the future. For example if we can get the number of remaining messages on each queue we could return the id of the worker with the least work etc.